ui/components/details.jsx

```diff
@@ -1,21 +1,21 @@
-var React = require('react');
+import React from 'react';
+import FailedTest from './failed.jsx';
+import PassedTest from './passed.jsx';
 
-var Details = React.createClass({
-  render: function() {
-    var paths = this.props.paths,
-        imgs = [<img className='baseline' src={paths.baseline}
-            key={paths.baseline}/>];
+class Details extends React.Component {
+  render() {
+    const paths = this.props.details.paths,
+        error = this.props.details.error;
```

Comment:
Indent this under the above var. I also know that we usually put individual and `const` for each variable but I don't know if that is mandatory /cc @andrei-picus-hs